pkg/kv/kvserver/testdata/flow_control_integration_v2/v1_to_v2_transition

```diff
@@ -0,0 +1,511 @@
+-- leader.
+SELECT range_id, store_id, crdb_internal.humanize_bytes(total_tracked_tokens::INT8)
+    FROM crdb_internal.kv_flow_control_handles_v2
+
+  range_id | store_id | total_tracked_tokens  
+-----------+----------+-----------------------
+
+
+-- Corresponding v2 token metrics on n2. All tokens should be returned.
+SELECT name, crdb_internal.humanize_bytes(value::INT8)
+    FROM crdb_internal.node_metrics
+   WHERE name LIKE '%kvflowcontrol%tokens%'
+ORDER BY name ASC;
+
+  kvflowcontrol.tokens.eval.elastic.available   | 24 MiB   
+  kvflowcontrol.tokens.eval.elastic.deducted    | 6.0 MiB  
+  kvflowcontrol.tokens.eval.elastic.returned    | 6.0 MiB  
+  kvflowcontrol.tokens.eval.elastic.unaccounted | 0 B      
+  kvflowcontrol.tokens.eval.regular.available   | 48 MiB   
+  kvflowcontrol.tokens.eval.regular.deducted    | 0 B      
+  kvflowcontrol.tokens.eval.regular.returned    | 0 B      
+  kvflowcontrol.tokens.eval.regular.unaccounted | 0 B      
+  kvflowcontrol.tokens.send.elastic.available   | 24 MiB   
+  kvflowcontrol.tokens.send.elastic.deducted    | 6.0 MiB  
+  kvflowcontrol.tokens.send.elastic.returned    | 6.0 MiB  
+  kvflowcontrol.tokens.send.elastic.unaccounted | 0 B      
+  kvflowcontrol.tokens.send.regular.available   | 48 MiB   
+  kvflowcontrol.tokens.send.regular.deducted    | 0 B      
+  kvflowcontrol.tokens.send.regular.returned    | 0 B      
+  kvflowcontrol.tokens.send.regular.unaccounted | 0 B      
+
+
+-- Viewing n1's v2 connected streams, there now should be three, as n1 acquired
+-- the leadership and lease.
+SELECT range_id, count(*) AS streams
+    FROM crdb_internal.kv_flow_control_handles_v2
+GROUP BY (range_id)
+ORDER BY streams DESC;
+
+  range_id | stream_count  
+-----------+---------------
+  70       | 3             
+
+
+-- (Issuing 1x1MiB regular, 3x replicated write that's not admitted.)
+
+
+-- The v2 tracked tokens per-stream on n1.
+SELECT range_id, store_id, crdb_internal.humanize_bytes(total_tracked_tokens::INT8)
+    FROM crdb_internal.kv_flow_control_handles_v2
+
+  range_id | store_id | total_tracked_tokens  
+-----------+----------+-----------------------
+  70       | 1        | 1.0 MiB               
+  70       | 2        | 1.0 MiB               
+  70       | 3        | 1.0 MiB               
+
+
+-- (Upgrading n2,n3 to use V2 encoding and protocol.)
+
+
+-- (Allow below-raft admission to proceed.)
+
+
+-- The v2 flow token metrics on n1.
+SELECT name, crdb_internal.humanize_bytes(value::INT8)
+    FROM crdb_internal.node_metrics
+   WHERE name LIKE '%kvflowcontrol%tokens%'
+ORDER BY name ASC;
+
+  kvflowcontrol.tokens.eval.elastic.available   | 24 MiB   
+  kvflowcontrol.tokens.eval.elastic.deducted    | 6.0 MiB  
+  kvflowcontrol.tokens.eval.elastic.returned    | 6.0 MiB  
+  kvflowcontrol.tokens.eval.elastic.unaccounted | 0 B      
+  kvflowcontrol.tokens.eval.regular.available   | 48 MiB   
+  kvflowcontrol.tokens.eval.regular.deducted    | 0 B      
```

Comment:
Why is there not 3 MiB regular tokens deducted.

Reply:
This was a test issue, querying the wrong node:

https://github.com/cockroachdb/cockroach/pull/130728/commits/ba7326c4234526cbb5b52e0daa7415004bd7034c#diff-07e30a242d2fa8847ebdaddf65fa5f505c0529b4ad8cc8b88a555f0ef99aca07R4188-R4189